features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/Landing/Menus/RestAPIMenu.jsx

```diff
@@ -8,12 +8,30 @@ import SampleAPI from 'AppComponents/Apis/Listing/SampleAPI/SampleAPI';
 import Divider from '@material-ui/core/Divider';
 import Box from '@material-ui/core/Box';
 import Configurations from 'Config';
+import { useApiListContext } from 'AppComponents/Shared/ApiListContext';
+
+/**
+ * Verify whether the sample API has already been deployed.
+ * @param {(Array | null)} apisAndApiProducts list of APIs
+ * @returns {boolean} true if API already deployed/exists
+ */
+const verifyIfSampleDeployed = (apisAndApiProducts) => {
+    if (apisAndApiProducts && Array.isArray(apisAndApiProducts)) {
+        return apisAndApiProducts.some((api) => api.name === 'PizzaShackAPI' && api.version === '1.0.0'
+            && api.context === '/pizzashack');
+    } else {
+        return false;
+    }
+};
 
 const RestAPIMenu = (props) => {
     const { icon, isCreateMenu } = props;
     const Component = isCreateMenu ? APICreateMenuSection : LandingMenu;
     const dense = isCreateMenu;
-    const { alwaysShowDeploySampleButton } = Configurations.apis;
+    const { alwaysShowDeploySampleButton = true } = Configurations.apis || {};
```

Comment:
This could have thrown an error if you load the page without the APIs config block, IMO if it's missing that means someone has tampered with the config file, so better to show an error, instead of defining a default value at the code level?  WDYT ?

Reply:
Agreed. fixed with 1bc0c49